jest.config.js

```diff
@@ -0,0 +1,24 @@
+// For a detailed explanation regarding each configuration property, visit:
```

Comment:
Why not QUnit?

Reply:
Looks like Jest is better for testing react apps and we don't need browser to test our new declarative components. So we decided to try using Jest